### DISCUSSION TITLE
Preparing for product release

### DISCUSSION THREAD

**Discussion by Chitusnair** [thawkes70/Tafe22S219#14] [General]
Hi Tony,

I have created a release branch and merged that with the development branch
Fixed the navigation of the MainPage in release branch. Merged the release branch to main branch and pushed to git repo.
![image](https://user-images.githubusercontent.com/29492335/190156485-4e1f41de-b49d-435e-9556-531df8663ccc.png)

**Comment by Chitusnair** [comment]
I have also checked out the master branch and created a tag with name V2.0

**Reply by Chitusnair** [reply]
![image](https://user-images.githubusercontent.com/29492335/190422816-c2e0b474-2889-4d6e-beb4-9385d4f73e71.png)

**Comment by thawkes70** [comment]
Hi Chitra,

Thanks for completing those sections.  Do you have the screen shots for section D?

**Reply by thawkes70** [reply]
I have attached the section B,D,G assessment document that I have completed so far, could you please review and add the bits for section D.   I hope I haven't made you fall behind because I was away.
[ICTICT449 - ASI - Assignment (Universal Calculator)_Assesment3.docx](https://github.com/thawkes70/Tafe22S219/files/9583677/ICTICT449.-.ASI.-.Assignment.Universal.Calculator._Assesment3.docx)

**Comment by Chitusnair** [comment]
Hi Tony,

I have added the screenshots for section D. Please let me know if I miss something.
[ICTICT449-ASI.-.Assignment.Universal.Calculator._Assesment3 (2).docx](https://github.com/thawkes70/Tafe22S219/files/9591773/ICTICT449-ASI.-.Assignment.Universal.Calculator._Assesment3.2.docx)

**Reply by thawkes70** [reply]
Looks good to me

**Comment by Chitusnair** [comment]
Hi,

Also, Can you please action the below steps from Section G?

![image](https://user-images.githubusercontent.com/29492335/190859390-f694cbfa-50bf-4792-a6d1-e7ff747fce53.png)
